arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -0,0 +1,2929 @@
+              ],
+              "description": "The inferred role of this process based on its name, command line, etc."
+            },
+            "details": {
+              "$ref": "#/definitions/ProcessDetails",
+              "description": "Process metadata (command line, product name, etc.)."
+            },
+            "user": {
+              "$ref": "#/definitions/ProcessUser",
+              "description": "Information about the account under which the process is executing."
+            },
+            "clientOf": {
+              "$ref": "#/definitions/ResourceReference",
+              "description": "Present only for a discovered process acting as a client of a monitored process/machine/port. References the monitored process/machine/port that this process is a client of."
+            },
+            "acceptorOf": {
+              "$ref": "#/definitions/ResourceReference",
+              "description": "Present only for a discovered process acting as a server. References the port on which the discovered process is accepting."
+            }
+          }
+        }
+      }
+    },
+    "Port": {
+      "description": "A port resource represents a server port on a machine. The port may be actively *monitored*, i.e., a Dependency Agent is running on its machine, or *discovered*, i.e., its existence was inferred by observing the data stream from monitored machines. A port is live during an interval of time, if that port had associated activity during (parts) of that interval.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CoreResource"
+        }
+      ],
+      "x-ms-discriminator-value": "port",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "Resource properties.",
+          "properties": {
+            "monitoringState": {
+              "$ref": "#/definitions/MonitoringState",
+              "description": "Specifies whether the port is actively monitored or discovered."
+            },
+            "machine": {
+              "$ref": "#/definitions/ResourceReference",
+              "description": "Machine hosting this port."
+            },
+            "displayName": {
+              "type": "string",
+              "description": "Name to use for display purposes."
+            },
+            "ipAddress": {
+              "type": "string",
+              "description": "IP address associated with the port. At present only IPv4 addresses are supported."
+            },
+            "portNumber": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Port number."
+            }
+          }
+        }
+      }
+    },
+    "ClientGroup": {
+      "description": "Represents a collection of clients of a resource. A client group can represent the clients of a port, process, or a machine.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CoreResource"
```

Comment:
I think maybe this is the wrong code segment to go with the comment about duration.  I agree that it's always better to be more clear.  However, this is not a part of the spec that changed in this PR.